checkov/github_actions/checks/job/CosignSBOM.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+from checkov.common.models.enums import CheckResult
+from checkov.github_actions.checks.base_github_action_check import BaseGithubActionsCheck
+from checkov.yaml_doc.enums import BlockType
+
+
+class CosignSignSBOM(BaseGithubActionsCheck):
+    def __init__(self):
+        name = "Find evidence of cosign sbom attestation in pipeline"
+        id = "CKV_GHA_6"
+        super().__init__(
+            name=name,
+            id=id,
+            block_type=BlockType.OBJECT,
+            supported_entities=['jobs']
+        )
+
+    def scan_entity_conf(self, conf):
+        jobs = conf.items()
+        for jobname,jobdetail in jobs:
+            if jobname == '__startline__':
```

Comment:
```suggestion
            if jobname == START_LINE:
```
also here, don't forget the import 😄